gatsby-node.js

```diff
@@ -99,4 +106,253 @@ async function makeSynoptic(createPage, reporter, graphql) {
       context: section
     })
   }
+}
+
+async function makeSearchIndex(reporter, graphql){
+  const lunr = require('lunr')
+  const remark = (await import('remark')).remark
```

Comment:
My preference is to import these modules at the top of the file. Is the dynamic import needed for `remark`?

Reply:
Yes, when I try to import remark like any other JS module, I get an error requesting me to change it to a dynamic import instead